optuna/logging.py

```diff
@@ -28,15 +28,18 @@
 _default_handler: Optional[logging.Handler] = None
 
 
-def create_default_formatter() -> colorlog.ColoredFormatter:
+def create_default_formatter() -> logging.Formatter:
     """Create a default formatter of log messages.
 
     This function is not supposed to be directly accessed by library users.
     """
-    return colorlog.ColoredFormatter(
-        "%(log_color)s[%(levelname)1.1s %(asctime)s]%(reset)s %(message)s",
-        no_color=False if _color_supported() else True,
-    )
+    header = "[%(levelname)1.1s %(asctime)s]"
+    message = "%(message)s"
+    if _color_supported():
+        return colorlog.ColoredFormatter(
+            f"%(log_color)s{header}%(reset)s {message}",
+        )
+    return logging.Formatter("[%(levelname)1.1s %(asctime)s] %(message)s")
```

Comment:
Thank you for your update. I think we can use `header` and `message` here.
```suggestion
    return logging.Formatter(f"{header} {message}")
```

Reply:
Oops, I forgot to change it. Thank you!